x/concentrated-liquidity/types/query.pb.go

```diff
@@ -302,58 +303,221 @@ func (m *QueryParamsResponse) GetParams() Params {
 	return Params{}
 }
 
+// =============================== LiquidityDepthsForRange
+type QueryLiquidityDepthsForRangeRequest struct {
+	PoolId     uint64                                 `protobuf:"varint,1,opt,name=pool_id,json=poolId,proto3" json:"pool_id,omitempty" yaml:"pool_id"`
+	LowerTick  github_com_cosmos_cosmos_sdk_types.Int `protobuf:"bytes,2,opt,name=lower_tick,json=lowerTick,proto3,customtype=github.com/cosmos/cosmos-sdk/types.Int" json:"lower_tick" yaml:"lower_tick"`
+	UpperTick  github_com_cosmos_cosmos_sdk_types.Int `protobuf:"bytes,3,opt,name=upper_tick,json=upperTick,proto3,customtype=github.com/cosmos/cosmos-sdk/types.Int" json:"upper_tick" yaml:"upper_tick"`
+	Pagination *query.PageRequest                     `protobuf:"bytes,4,opt,name=pagination,proto3" json:"pagination,omitempty"`
+}
+
+func (m *QueryLiquidityDepthsForRangeRequest) Reset()         { *m = QueryLiquidityDepthsForRangeRequest{} }
+func (m *QueryLiquidityDepthsForRangeRequest) String() string { return proto.CompactTextString(m) }
+func (*QueryLiquidityDepthsForRangeRequest) ProtoMessage()    {}
+func (*QueryLiquidityDepthsForRangeRequest) Descriptor() ([]byte, []int) {
+	return fileDescriptor_ce34c1e206115391, []int{6}
+}
+func (m *QueryLiquidityDepthsForRangeRequest) XXX_Unmarshal(b []byte) error {
+	return m.Unmarshal(b)
+}
+func (m *QueryLiquidityDepthsForRangeRequest) XXX_Marshal(b []byte, deterministic bool) ([]byte, error) {
+	if deterministic {
+		return xxx_messageInfo_QueryLiquidityDepthsForRangeRequest.Marshal(b, m, deterministic)
+	} else {
+		b = b[:cap(b)]
+		n, err := m.MarshalToSizedBuffer(b)
+		if err != nil {
+			return nil, err
+		}
+		return b[:n], nil
+	}
+}
+func (m *QueryLiquidityDepthsForRangeRequest) XXX_Merge(src proto.Message) {
+	xxx_messageInfo_QueryLiquidityDepthsForRangeRequest.Merge(m, src)
+}
+func (m *QueryLiquidityDepthsForRangeRequest) XXX_Size() int {
+	return m.Size()
+}
+func (m *QueryLiquidityDepthsForRangeRequest) XXX_DiscardUnknown() {
+	xxx_messageInfo_QueryLiquidityDepthsForRangeRequest.DiscardUnknown(m)
+}
+
+var xxx_messageInfo_QueryLiquidityDepthsForRangeRequest proto.InternalMessageInfo
+
+func (m *QueryLiquidityDepthsForRangeRequest) GetPoolId() uint64 {
+	if m != nil {
+		return m.PoolId
+	}
+	return 0
+}
+
+func (m *QueryLiquidityDepthsForRangeRequest) GetPagination() *query.PageRequest {
+	if m != nil {
+		return m.Pagination
+	}
+	return nil
+}
+
+type QueryLiquidityDepthsForRangeResponse struct {
+	LiquidityDepths []LiquidityDepth    `protobuf:"bytes,1,rep,name=liquidity_depths,json=liquidityDepths,proto3" json:"liquidity_depths"`
+	Pagination      *query.PageResponse `protobuf:"bytes,2,opt,name=pagination,proto3" json:"pagination,omitempty"`
```

Comment:
What is this pagination object used for if we're not using pagination? Is this pagination within the same range?

RE:
https://github.com/osmosis-labs/osmosis/pull/4167#issuecomment-1411488545

Reply:
this has been removed